build/contrib/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:14.04
+
+RUN \
+    apt-get update && \
+    apt-get -y upgrade && \
+    apt-get install -y python-pip python-setuptools bridge-utils \
+                       libglib2.0-dev libdbus-1-dev libudev-dev \
+                       libical-dev libreadline-dev git-core && \
+    pip install --user grpc google-cloud protobuf && \
+    sudo cpan -i Text::Template && \
+    mkdir weave
+
+COPY . weave
+
+RUN cd weave && \
+    ./configure --disable-systemd && \
+    make all && \
+    make install
```

Comment:
I think we should be using Makefile-Standalone whenever we build any of the desktop tools, rather than invoking ./configure with default args.  The former employs project configuration files that configures OpenWeave specifically for desktop use.